### DISCUSSION TITLE
Deploy ejbca docker image to server

### DISCUSSION THREAD

**Discussion by ThanhNam20** [Keyfactor/ejbca-ce#117] [Q&A]
Hi,
I'm new to Docker and  EJBCA. I installed EJBCA docker-image following this [one](https://hub.docker.com/r/keyfactor/ejbca-ce) and working in my local. But I want to run it on my VPS server and can access into Admin Gui with my server ip. Please help 🙏,
Thanks.

**Comment by primetomas** [comment]
I'm not sure how your VPS server works, but here are a few resources for different cloud/internal cloud deployment options. VMs and Kubernetes.
https://github.com/Keyfactor/ansible-ejbca-signserver-playbooks
https://github.com/Keyfactor/ejbca-ce-helm
https://github.com/Keyfactor/ejbca-containers
